data/event-schemas/schema/m.room.power_levels.yaml

```diff
@@ -5,6 +5,8 @@ description: |-
   This event specifies the minimum level a user must have in order to perform a
   certain action. It also specifies the levels of each user in the room.
 
+  The allowed range of power levels is specified by the [Canonical JSON specification](/appendices/#canonical-json).
```

Comment:
Also, this is true for every numeric in the protocol, and the current phrasing makes it sound like it's specific to PLs.

Also also: making people go and grok the canonical-json spec rather than just telling them the answer seems... unhelpful?

I would phrase it as something like:

> Note: the allowed range for power level values is `[-(2**53)+1, (2**53)-1]`, as specified by the [Canonical JSON specification](/appendices/#canonical-json).

Reply:
I've removed the actual values because of https://github.com/matrix-org/matrix-spec/pull/1169#discussion_r923618961. Personally I'd also prefer the values, but I can also see that potentially outdated values might be seen as an issue?

Other than that, I'll move it to the end and look into rephrasing so it doesn't sound specific for PLs.